setup.py

```diff
@@ -122,7 +121,7 @@ def write_version_file(version, git_revision):
     git_revision : str
         The full commit hash for the current Git revision.
     """
-    with io.open(VERSION_FILE, "w", encoding="ascii") as version_file:
+    with open(VERSION_FILE, "w", encoding="ascii") as version_file:
```

Comment:
Any particular reason for `ascii` here as opposed to `utf-8` (`ascii` made sense when we still had to support Python 2, but now I think `utf-8` would be preferred for auto-generated source files).

Reply:
Yep, let's switch it to `utf-8`.